beekeeper-path-scheduler-apiary/src/test/java/com/expediagroup/beekeeper/scheduler/filter/EventTypeListenerEventFilterTest.java

```diff
@@ -46,36 +57,36 @@
 
   @Test
   public void typicalFilterAlterPartitionEvent() {
-    when(alterPartitionEvent.getEventType()).thenReturn(EventType.ALTER_PARTITION);
-    boolean filter = listenerEventFilter.filter(alterPartitionEvent, UNREFERENCED);
+    when(this.alterPartitionEvent.getEventType()).thenReturn(ALTER_PARTITION);
```

Comment:
Don't think you need all the `this`

Reply:
Yh sorry. It was this plugin I had which automatically changed these files when I opened them. I have disabled it now though